packages/account/src/Helpers/utils.ts

```diff
@@ -66,7 +68,7 @@ export const getDocumentData = (country_code: string, document_type: string) =>
 };
 
 export const preventEmptyClipboardPaste = (e: React.ClipboardEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-    const clipboardData = (e.clipboardData ?? window.clipboardData).getData('text');
+    const clipboardData = (e.clipboardData ?? (window as any).clipboardData).getData('text');
```

Comment:
@yauheni-deriv   one more nit suggestion :) we can declare this on top of this file:
```
declare global {
    interface Window { 
         clipboardData: DataTransfer
    }
}
```
and then we won't need to add `any` to window :)

Reply:
thanks 👍